src/components/Login/Login.js

```diff
@@ -60,7 +60,11 @@ const Login = ({onLoginClick, disabled}) => {
                     <LockOutlinedIcon />
                 </Avatar>
                 <Typography component="h1" variant="h5">
-                    <FormattedMessage id="login/login" defaultMessage={"login"}/> ?
+                    <FormattedMessage
+                        id="login/login"
+                        defaultMessage={'login'}
+                    />{' '}
```

Comment:
{' '} ?

Reply:
good catch

Reply:
actually necessary to maintain the whitespace between the year and the question mark